lib/output.js

```diff
@@ -597,6 +598,12 @@ function OutputStream(options) {
             return true;
         }
 
+        if (output.option('webkit')
+            && output.parent(0) instanceof AST_PropAccess
+            && output.parent(1) instanceof AST_Assign) {
```

Comment:
> now I'm getting failure from pdfjs.js in test/jetstream.js - investigating.

If your patch with the AST_Assign condition works - just use it.

Can the `webkit` output option be hardcoded in `test/jetstream.js` by default?